keyboards/akb/ogr/readme.md

```diff
@@ -0,0 +1,29 @@
+# akb/ogr
+
+![akb/ogr](https://alchemistkeyboards.com/wp-content/uploads/2022/02/EPUUbTT.png)
```

Comment:
Please rehost this on Imgur or the like.

Reply:
resolved. thank you